packages/quantic/force-app/main/default/lwc/quanticAriaLive/quanticAriaLive.js

```diff
@@ -0,0 +1,32 @@
+import { LightningElement, track, api } from 'lwc';
+
+export default class QuanticAriaLive extends LightningElement {
+  /** @type {Object.<string, Object>} */
```

Comment:
You could define the interface above and use it here instead of Object.

Example:
```ts
/**
 * @typedef AriaLiveRegion
 * @property {string} message
 * @property {boolean} assertive
 */
```
```suggestion
  /** @type {Record<string, AriaLiveRegion>} */
```